source/ed/ed2nav.cpp

```diff
@@ -165,6 +165,33 @@ struct FindAdminWithCities {
     }
 };
 
+bool write_data_to_file(const std::string& output_filename, navitia::type::Data& data) {
+    auto logger = log4cplus::Logger::getInstance("log");
+    std::string temp_output = output_filename + ".temp";
+    try {
+        data.save(temp_output);
+        try {
+            remove(output_filename.c_str());
```

Comment:
You should probably rename it first.

Reply:
I see what you mean but if I rename `temp_output` file name to `output_filename`, and then remove the `output_filename` file, wouldn't there be a problem ?

Reply:
What I meant was to rename the current `data.nav.lz4` to something like `data.nav.lz4.bak`, then rename `data.nav.lz4.temp`  to `data.nav.lz4`, this way if the renaming fail for whatever reason we still have a valid dataset to recover.